livechat/customer/web/client.py

```diff
@@ -998,14 +1013,163 @@ def cancel_greeting(self,
+                                however, these method-level parameters will not be persisted across requests.
+
+            Returns:
+                httpx.Response: The Response object from `httpx` library,
+                                which contains a server’s response to an HTTP request. '''
+        if payload is None:
+            payload = {}
+        params = {}
+        if namespace:
+            params['namespace'] = namespace
+        if name:
+            params['name'] = name
+        params['license_id'] = self.license_id
+        return self.session.post(f'{self.api_url}/list_license_properties',
+                                 json=payload,
+                                 params=params,
+                                 headers=headers)
+
+    def list_group_properties(self,
+                              group_id: int = None,
+                              namespace: str = None,
+                              name: str = None,
+                              payload: dict = None,
+                              headers: dict = None) -> httpx.Response:
+        ''' Returns the properties of a given group. It only returns the properties a Customer has access to.
+            Args:
+                group_id (int): ID of the group you want to return the properties of.
+                namespace (str): Property namespace to retrieve.
+                name (str): Property name.
+                payload (dict): Custom payload to be used as request's data.
+                headers (dict): Custom headers to be used with session headers.
+                                They will be merged with session-level values that are set,
+                                however, these method-level parameters will not be persisted across requests.
+
+            Returns:
+                httpx.Response: The Response object from `httpx` library,
+                                which contains a server’s response to an HTTP request. '''
+        if payload is None:
+            payload = {}
+        params = {}
+        if namespace:
+            params['namespace'] = namespace
+        if name:
+            params['name'] = name
+        if group_id:
+            params['id'] = str(group_id)
+        params['license_id'] = self.license_id
+        return self.session.post(f'{self.api_url}/list_group_properties',
+                                 json=payload,
+                                 params=params,
+                                 headers=headers)
 
 
 class CustomerWeb34(CustomerWebInterface):
     ''' Customer API version 3.4 class. '''
+    def __init__(self, organization_id: str, access_token: str, version: str,
+                 url: str, http2: bool) -> CustomerWeb34:
+        super().__init__(access_token, version, url, http2)
+        if all([organization_id, isinstance(organization_id, str)]):
+            self.organization_id = organization_id
+            self.query_string = f'?organization_id={str(organization_id)}'
```

Comment:
Is is necessary to convert it to string (`str()`) considering previous `isinstance(organization_id, str)`?